src/rdkafka_conf.h

```diff
@@ -237,6 +238,12 @@ struct rd_kafka_conf_s {
                 char *ca_location;
                 rd_kafka_cert_t *ca;
                 char *crl_location;
+#if OPENSSL_VERSION_NUMBER >= 0x10100000
+                ENGINE *engine;
+                char   *engine_location;
+                char   *engine_id;
+                void   *engine_callback_data;
```

Comment:
The rdkafka.h API had this as const, why is not non-const here?

Reply:
removed "const" for callback_data across since openssl needs it non-const callback_data
https://docs.huihoo.com/doxygen/openssl/1.0.1c/eng__pkey_8c.html#a9162e0cbce47fc37d548df130a2ba4c5